src/sqlancer/postgres/oracle/tlp/PostgresTLPBase.java

```diff
@@ -65,7 +65,12 @@ public void check() throws SQLException {
     }
 
     List<PostgresExpression> generateFetchColumns() {
-        return Arrays.asList(new PostgresColumnValue(targetTables.getColumns().get(0), null));
+        List<PostgresExpression> fetchColumns = new ArrayList<>();
```

Comment:
I think you could copy the structure in https://github.com/sqlancer/sqlancer/blob/master/src/sqlancer/sqlite3/oracle/tlp/SQLite3TLPBase.java#L58 to have a column named `*` to implement a `SELECT *`. Like this, we can still the detect the case where the columns were erroneously not set.